packagesDev/next-config/dist/generated/config.js

```diff
@@ -1,110 +1,77 @@
+        productFiltersPro: zod_1.z.boolean().nullish(),
+        productRoute: zod_1.z.string().nullish(),
+        robotsAllow: zod_1.z.boolean().nullish(),
+        storefront: zod_1.z.array(GraphCommerceStorefrontConfigSchema()),
+        wishlistHideForGuests: zod_1.z.boolean().nullish(),
+        wishlistIgnoreProductWishlistStatus: zod_1.z.boolean().nullish(),
+        wishlistShowFeedbackMessage: zod_1.z.boolean().nullish()
     });
 }
+exports.GraphCommerceConfigSchema = GraphCommerceConfigSchema;
 function GraphCommerceDebugConfigSchema() {
-    return _zod.z.object({
-        pluginStatus: _zod.z.boolean().nullish(),
-        webpackCircularDependencyPlugin: _zod.z.boolean().nullish(),
-        webpackDuplicatesPlugin: _zod.z.boolean().nullish()
+    return zod_1.z.object({
+        pluginStatus: zod_1.z.boolean().nullish(),
+        webpackCircularDependencyPlugin: zod_1.z.boolean().nullish(),
+        webpackDuplicatesPlugin: zod_1.z.boolean().nullish()
     });
 }
+exports.GraphCommerceDebugConfigSchema = GraphCommerceDebugConfigSchema;
 function GraphCommerceStorefrontConfigSchema() {
-    return _zod.z.object({
-        canonicalBaseUrl: _zod.z.string().nullish(),
-        cartDisplayPricesInclTax: _zod.z.boolean().nullish(),
-        defaultLocale: _zod.z.boolean().nullish(),
-        domain: _zod.z.string().nullish(),
-        googleAnalyticsId: _zod.z.string().nullish(),
-        googleRecaptchaKey: _zod.z.string().nullish(),
-        googleTagmanagerId: _zod.z.string().nullish(),
-        hygraphLocales: _zod.z.array(_zod.z.string().min(1)).nullish(),
-        linguiLocale: _zod.z.string().nullish(),
-        locale: _zod.z.string().min(1),
-        magentoStoreCode: _zod.z.string().min(1)
+    return zod_1.z.object({
+        canonicalBaseUrl: zod_1.z.string().nullish(),
+        cartDisplayPricesInclTax: zod_1.z.boolean().nullish(),
+        defaultLocale: zod_1.z.boolean().nullish(),
+        domain: zod_1.z.string().nullish(),
+        googleAnalyticsId: zod_1.z.string().nullish(),
+        googleRecaptchaKey: zod_1.z.string().nullish(),
+        googleTagmanagerId: zod_1.z.string().nullish(),
+        hygraphLocales: zod_1.z.array(zod_1.z.string().min(1)).nullish(),
+        linguiLocale: zod_1.z.string().nullish(),
+        locale: zod_1.z.string().min(1),
+        magentoStoreCode: zod_1.z.string().min(1)
     });
 }
+exports.GraphCommerceStorefrontConfigSchema = GraphCommerceStorefrontConfigSchema;
 function MagentoConfigurableVariantValuesSchema() {
-    return _zod.z.object({
-        content: _zod.z.boolean().nullish(),
-        gallery: _zod.z.boolean().nullish(),
-        url: _zod.z.boolean().nullish()
+    return zod_1.z.object({
+        content: zod_1.z.boolean().nullish(),
+        gallery: zod_1.z.boolean().nullish(),
+        url: zod_1.z.boolean().nullish()
     });
 }
+exports.MagentoConfigurableVariantValuesSchema = MagentoConfigurableVariantValuesSchema;
```

Comment:
Always run `yarn codegen` in the example dir after using `yarn packages` in the root